charts/vela-core/templates/defwithtemplate/gateway.yaml

```diff
@@ -125,6 +133,12 @@ spec:
 
         	// +usage=Specify a pathType for the ingress rules, defaults to "ImplementationSpecific"
         	pathType: *"ImplementationSpecific" | "Prefix" | "Exact"
+
+        	// +usage=Specify the annotations to be added to the gateway
+        	annotations?: [string]: string | null
```

Comment:
since annotations and labels are array in K8s, why the parameter type is not array?